controls/overlay.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+overlay_controls = input('overlay_controls')
+system_categorization = input('system_categorization')
+
+include_controls 'redhat-enterprise-linux-8-stig-baseline' do
+  unless overlay_controls.empty?
+    overlay_controls.each do |overlay_control|
+      control overlay_control do
+        impact 0.0
+        desc "caveat", "Not applicable for this CMS ARS 5.0 overlay, since the requirement is not included in CMS ARS 5.0 for the system categorization of #{system_categorization}"
+      end
+    end
+  end
+
+  
+  ## NIST tags updated due to changes between NIST SP 800-53 rev 4 and rev 5 (https://csrc.nist.gov/csrc/media/Publications/sp/800-53/rev-5/final/documents/sp800-53r4-to-r5-comparison-workbook.xlsx)
+
+  ## IA-2(6) incorporates withdrawn controls IA-2(7) and IA-2(11)
+
+  control 'SV-230273' do
+    tag nist: ["IA-2 (6)"]
+  end
+  
+  control 'SV-230274' do
+    tag nist: ["IA-2 (6)"]
+  end
+
+  ## SC-45(1) incorporates withdrawn control AU-8(1)
+  
+  control 'SV-230484' do
+    tag nist: ["SC-45 (1)"]
+  end
+  
+  ## AC-2(3) maintains requirement to disable accounts, removed from rev 4's IA-5(1)"(e)"
```

Comment:
Should it be `removed from rev 4's IA-4 e`?

Reply:
Done https://github.com/CMSgov/cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay/pull/1/commits/bc07c2819aa1c7216c716f04e9075133079dbc26